### PR TITLE
Work Page

### PR DESCRIPTION
**Pull request by IfzaRasool**
Work Page is a recent post page where you upload your post with a post description.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -75,6 +76,95 @@ <h2 class="secondry-content">Software Developer</h2>
...
+
+     <div class="work-section-wrap">
+         
+        <div class="arrow-line">
+            <img src="Icons/arrow-bar.png" alt="arrow line">
+        </div>
+        <div class="arrow-head">
+            <img src="Icons/arrow-head.png" alt="arrow head">
+        </div>
+        <h2 class="work-text">Work</h2>
```
Comment (emmanuellekamwa) [inline index.html:91]: - [ ] The list of social medias and the work title are not centrally aligned. Please try and fix that.

**Review by emmanuellekamwa — CHANGES_REQUESTED** [review]
**STATUS: CHANGES REQUESTED ❌⛔**
Hello @IfzaRasool 👋
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._
- Your readme file does not really look presentable. Please check this [link](https://github.com/microverseinc/readme-template) so as to do a good readme.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by lily-coder — APPROVED** [review]
Hi @IfzaRasool , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -75,6 +76,95 @@ <h2 class="secondry-content">Software Developer</h2>
...
+                    <li>bootstrap</li>
+                    <li> <img src="Icons/Rectangle.png" alt="divide line3"></li>
+                    <li>Ruby</li>
+                   </ul>
+                <div class="button"><img src="Icons/seeproject.png" alt="button"></div>
+            </section>
+
+
+
+        </div>
```
[the remark this answers is not in the document]
Comment (IfzaRasool) [reply index.html:168]: ### ok

[the remark this answers is not in the document]
Comment (IfzaRasool) [reply index.html:168]: > ssss

[the remark this answers is not in the document]
Comment (IfzaRasool) [reply index.html:168]: > ssssssssssssss``

[the remark this answers is not in the document]
Comment (IfzaRasool) [reply index.html:168]: sssssssssssss

- [ ] ssss

**Pull request merged**